website/docs/operator/data-recovery.md

```diff
@@ -4,4 +4,288 @@ title: Data Recovery
 description: Provides documentation for Operators to recover missing data.
 ---
 
-This document in under construction.
+import MyImage from '@site/docs/templates/pan-zoom-image.mdx';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+## Recovery via Cumulus Dashboard
+
+Recovery processes are kicked off manually by an operator through the Cumulus Dashboard. 
+The dashboard calls an API which kicks off a recovery workflow.
+Recovery is an asynchronous operation since data
+requested from GLACIER can take up to 4 hours or more to reconstitute,
+and DEEP_ARCHIVE can take 12 hours. 
+Since it is asynchronous, the recovery container
+relies on a database to maintain the status of the request and event
+driven triggers to restore the data once it has been reconstituted
+from archive into an S3 bucket. Currently, data is copied back to the
+Cumulus S3 primary data bucket which is the default bucket. The operator 
+has the option to override the default bucket with another restore bucket if desired. 
+Determining the status of the recovery job is done manually by querying the database
+directly or by checking the status on the dashboard.
+
+A screenshot of the Cumulus dashboard used for recovering granules is shown below.
+
+<MyImage
+    imageSource={useBaseUrl('img/Cumulus-Dashboard-Recovery-Workflow.png')}
+    imageAlt="Cumulus Dashboard used for recovery"
+    zoomInPic={useBaseUrl('img/zoom-in.svg')}
+    zoomOutPic={useBaseUrl('img/zoom-out.svg')}
+    resetPic={useBaseUrl('img/zoom-pan-reset.svg')}
+/>
+
+On the dashboard home page, click on `Granules` option and add the granule to recover.
+Then click on the `Options` button and select `Execute`. From the dropdown menu, 
+select `OrcaRecoveryWorkflow` and hit `Confirm`. This will execute the recovery process.
+There are several configurable parameters that can be set up while running the workflow and are explained in 
+the section [below](#recovery-workflow-configurable-parameters).
+
+## Manual recovery via step function workflow
+
+An operator can also run the recovery by running the `Recovery Workflow` in step function. 
+
+:::warning
+The operator should have access to AWS console to manually run the workflow which is not ideal.
+:::
+
+### Recovery workflow input and output examples
+
+The following is an input event example that an operator might set up while running the recovery workflow.
+
+```json
+{
```

Comment:
Minor bit of indentation formatting here.

Reply:
fixed